lib/nuntiux.ex

```diff
@@ -11,14 +11,18 @@ defmodule Nuntiux do
   @type history :: Nuntiux.Mocker.history()
   @type received? :: Nuntiux.Mocker.received?()
   @type event :: Nuntiux.Mocker.event()
+  @type expect_fun :: Nuntiux.Mocker.expect_fun()
+  @type expect_name :: Nuntiux.Mocker.expect_name()
+  @type expect_id :: Nuntiux.Mocker.expect_id()
+  @type expects :: Nuntiux.Mocker.expects()
```

Comment:
Still not sure if this style of exporting types (from another module) will satisfy me in the end, but I'll keep doing it until I refactor some stuff at the end, like we did for `nuntius`.

Reply:
This almost feels like Elixir's missing a `typedelegate` :)